build.gradle.kts

```diff
@@ -84,3 +86,27 @@ publishing {
         }
     }
 }
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        // New versions of ktlint display:
+        // "Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files."
+        // but .editorconfig file is ignored (Similar issue: https://github.com/pinterest/ktlint/issues/1599 )
+        ktlint("0.46.0").editorConfigOverride(
+            mapOf(
+                "disabled_rules" to "package-name"
+            )
+        )
+    }
+    kotlinGradle {
+        target("*.gradle.kts")
+        ktlint().editorConfigOverride(
+            mapOf("indent_size" to "4")
+        )
+    }
+}
+
+gitHooks {
+    setHooks(mapOf("pre-commit" to "spotlessCheck"))
+    setHooksDirectory(layout.projectDirectory.dir("/tmp/tests-execution-chart/.git/hooks"))
```

Comment:
Setting hooks directory to `/tmp/tests-execution-chart/.git/hooks` will not result with working git hook. We should not set `setHooksDirectory`. By default hooks will be created in .git/hooks and it is ok.

Reply:
It seemed to me working. I tested with the default location first, then changed it but not removed the first hook :-(
setHooksDirectory removed